docs/appendix/converting_from_20.rst

```diff
@@ -0,0 +1,173 @@
+###########################################################
+Changes and Considerations Converting from NVIDIA FLARE 2.0
+###########################################################
+There are several breaking changes:
+
+Provisioning Changes
```

Comment:
all these new components and concepts are introduced in terms of 2.0 -> 2.1.0 conversion. 
But for new users, these concepts are hidden and not exposed. 
I suggest to have a new section under Programming_Guide, something like
* programming_guide
 **  system architecture
 **  concepts and system components 
 ** events and event handles
 ** 

Then, in the current 2.0 -> 2.1 conversion section, you can simply reference the link to the "Concept and System components".